src/content/app/tools/blast/components/blast-job-submit/BlastJobSubmit.tsx

```diff
@@ -15,17 +15,39 @@
  */
 
 import React from 'react';
-import noop from 'lodash/noop';
+import { useSelector } from 'react-redux';
 
 import { PrimaryButton } from 'src/shared/components/button/Button';
 
+import { getBlastFormData } from '../../state/blast-form/blastFormSelectors';
+import { BlastFormState } from '../../state/blast-form/blastFormSlice';
+
 const BlastJobSubmit = () => {
   // TODO:
   // 1) validate the blast form before enabling the button
   // 2) actually do the job submission
+  const blastFormData = useSelector(getBlastFormData);
+
+  const onBlastSubmit = () => {
+    const postData = createBlastSubmissionData(blastFormData);
+    return postData;
+  };
+
+  const createBlastSubmissionData = (blastFormData: BlastFormState) => {
+    return {
+      email: 'blast2020@ebi.ac.uk',
+      genomeIds: blastFormData.selectedSpecies,
+      program: blastFormData.settings.program,
+      parameters: {
+        title: blastFormData.settings.jobName,
+        database: blastFormData.settings.sequenceType,
```

Comment:
No, database is different from the sequence type.

<img width="457" alt="image" src="https://user-images.githubusercontent.com/6834224/155244784-743fd6f7-be1f-47b0-863f-dea8055022ce.png">

Database values (`dna`, `cdna` and `pep` as suggested by Andres) come from the blast config (parameters→database). It is the config file that needs to be updated.

Sequence type is submitted as the `stype` param, and is dna or protein (see links to docs in another comment)

Reply:
yes true. will change it. sleepy head

Reply:
The [input config](https://github.com/Ensembl/ensembl-client/blob/dev/src/content/app/tools/blast/components/blast-settings/blastSettingsConfig.json) should also be updated (currently has "dna/cdna/protein")